src/test/java/com/google/javascript/clutz/partial/to_object.d.ts

```diff
@@ -0,0 +1,14 @@
+declare namespace ಠ_ಠ.clutz.goog.array {
+  /**
```

Comment:
can you simplify the test to the its core issue. You can point to the goog.array.toObject in a comment https://github.com/google/closure-library/blob/master/closure/goog/array/array.js#L22

Reply:
Done.